File: FizzBuzz Solutions/fizzbuzz.rs
fn fizzbuzz(limit: i32){
  for int in 1..limit{
    if int % 15 == 0{
      println!("FizzBuzz");
    }
    else if int % 5 == 0{
      println!("Buzz");
    }
    else if int % 3 == 0{
      println!("Fizz");
    }
    else{
      println!("{}", int);
    }
  }
}

fn main() {
  fizzbuzz(100)
}

File: FizzBuzz Solutions/fizzbuzz.php
<?php
function fizzBuzz($limit = 100){
  for ($i = 1; $i <= $limit; $i++) {
    if ($i % 15 == 0) {
        echo 'FizzBuzz <br>';
    } elseif ($i % 3 == 0) {
        echo 'Fizz <br>';
    } elseif ($i % 5 == 0) {
        echo 'Buzz <br>';
    } else {
        echo $i . '<br>';
    }
  }
}

fizzBuzz(100);

?>
